Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,27 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.
+let age = 32;
+let position = 'Product Owner';
+let priorExperience = true; 
+let otherDetails = null; 
+
+class User {
+  constructor(first, last) {
+    this.firstName = first;
+    this.lastName = last;
+  }
+}
+let candidate = new User ('Michael', 'Jacobson'); //3. Demonstrate object's references.
+
+console.log(candidate,age,position,priorExperience,otherDetails);
```

Comment:
we will talk about that on lesson, it is not showing `object`'s references currently